src/Middleware/WebSockets/src/WebSocketsDependencyInjectionExtensions.cs

```diff
@@ -11,6 +11,11 @@ public static class WebSocketsDependencyInjectionExtensions
     {
         public static IServiceCollection AddWebSockets(this IServiceCollection services, Action<WebSocketOptions> configure)
         {
+            if (configure is null)
```

Comment:
`services.Configure` will throw if `configure` is null. Might as well throw it here and make it consistent with all of our other extension methods.